beams/service/rpc_client.py

```diff
@@ -167,18 +167,25 @@ def run(self, command: str, **kwargs) -> HeartBeatReply:
             response = self.get_heartbeat()
             return response
 
+        # TODO: deal with none as tree name
+        tree_name = kwargs.get("tree_name") or ""
+        tree_uuid = kwargs.get("tree_uuid") or ""
+        # These commands only need captured tree name and command itself
+        if command.upper() == "GET_TREE_DETAILS":
+            response = self.get_detailed_update(tree_name=tree_name, tree_uuid=tree_uuid)
+            return response
```

Comment:
Nitpick: unused variable
```suggestion
            return self.get_detailed_update(tree_name=tree_name, tree_uuid=tree_uuid)
```